src/redux/consigureStore.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Please do not disable linters, fixing linters errors help programmers write a good and clean codebase. Kindly fix that 😉 🙃